topics/assembly/tutorials/flye-assembly/tutorial.md

```diff
@@ -51,144 +52,141 @@ In this tutorial, we will assemble a genome of a species of fungi in the family
 >
-> 1. Create a JGI account in registration page of JGI: [JGI registration](https://contacts.jgi.doe.gov/registration/new)
-> 2. Sign in JGI Genome Portal [JGI Genome Portal](https://genome.jgi.doe.gov/portal/)
-> 3. Genome assembly is available here: [JGI Mucor mucedo](https://genome.jgi.doe.gov/portal/Mucmuc1/Mucmuc1.download.html)
-> 4. Import fasta assembly file `Mucmuc1_AssemblyScaffolds.fasta` on your computer locally
-> 5. Upload this file on Galaxy
-> 6. Check that the datatype is `fasta`
+> 1. Reference genome is available here: [ASM4765177v1 assembly for Aspergillus niger](https://www.ebi.ac.uk/ena/browser/view/GCA_047651775.1)
+> 2. Download the `WGS Set FASTA (JBKZXA01.fasta.gz)` on your computer
+> 3. Upload this file on Galaxy
+> 4. Check that the datatype is `fasta.gz`
 >
 >    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
 >
 {: .hands_on}
 
-# Genome Assembly with **Flye**
+# Genome Assembly
 
-We will use *Flye*, a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PacBio / ONT reads as input and outputs polished contigs. Flye also has a special mode for metagenome assembly. All informations about Flye assembler are here: [Flye](https://github.com/fenderglass/Flye/).
+{% include _includes/cyoa-choices.html option1="Hifiasm" option2="Flye" default="Hifiasm"
+       text="[Hifiasm](https://github.com/chhylp123/hifiasm) and [Flye](https://github.com/mikolmogorov/Flye) are two well known assembler." %}
+<div class="Hifiasm" markdown="1">
+
+Hifiasm is a fast haplotype-resolved de novo assembler initially designed for PacBio HiFi reads. In general, hifiasm generates the assembly graphs in the GFA format, so a step of conversion to fasta is necessary. The GFA 1.0 format is a tab-delimited text format for describing a set of sequences and their overlap.
+<br>
+Hifiasm produces arguably the best single-sample telomere-to-telomere assemblies combing HiFi, ultralong and Hi-C reads, and it is one of the best haplotype-resolved assemblers for the trio-binning assembly given parental short reads. For a human genome, hifiasm can produce the telomere-to-telomere assembly in one day.
 
-> <hands-on-title>Assembly</hands-on-title>
+> <hands-on-title>Assembly with Hifiasm</hands-on-title>
 >
-> 1. {% tool [Flye](toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye/2.9+galaxy0) %} with the following parameters:
->    - {% icon param-file %} *"Input reads"*: the three sequencing datasets
->    - *"Mode"*: `PacBio raw`
->    - *"Number of polishing iterations"*: `1`
->    - *"Reduced contig assembly coverage"*: `Disable reduced coverage for initial disjointing assembly`
+> 1. {% tool [Hifiasm](https://toolshed.g2.bx.psu.edu/repos/bgruening/hifiasm) %} with the following parameters:
```

Comment:
It is better to include here tool version as well. This way the training can not suddenly break.'

In your Galaxy instance on the top of you toolform you can find the tool version link

![grafik](https://github.com/user-attachments/assets/cf4db988-7c03-40e9-b00a-247001e17186)


remove the `https://usegalaxy.eu/root?tool_id=` from the begining.

Reply:
Done, thanks!